common/users/permissions.ts

```diff
@@ -13,3 +13,19 @@ export interface SbPermissions {
 }
 
 export type PermissionName = keyof SbPermissions
+
+export const DEFAULT_PERMISSIONS: Readonly<SbPermissions> = {
+  editPermissions: false,
+  debug: false,
+  banUsers: false,
+  manageLeagues: false,
+  manageMaps: false,
+  manageMapPools: false,
+  manageMatchmakingSeasons: false,
+  manageMatchmakingTimes: false,
+  manageRallyPointServers: false,
+  massDeleteMaps: false,
+  moderateChatChannels: false,
+}
+
+export const userPermissions = { ...DEFAULT_PERMISSIONS }
```

Comment:
huh? why is this here now?

Reply:
And when you move this back into the test file where it belongs, do *NOT* put this at the top of the file. When you do that you're contaminating every test if you modify it.

The point of `DEFAULT_PERMISSIONS` being readonly was to make sure that never happens, if you immediately turn it into a top-level non-const thing you have defeated the whole purpose